forwarding/test_match_host_forwarded.py

```diff
@@ -0,0 +1,293 @@
+"""
+Tests for verifying correctness of matching
+all host headers (URI, Host, Forwarded).
+"""
+from helpers import chains
+from framework import tester
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2022 Tempesta Technologies, Inc.'
```

Comment:
You're not a member of Tempesta Technologies, so I suggest to add your own authorship and copyrights in both the files. Also feel free to add you copyright lines to any of the modified files in https://github.com/tempesta-tech/tempesta/pull/1629